app/reports_generation/scripts/utils.py

```diff
@@ -25,3 +28,20 @@ def validate_is_number(config: dict, key: str):
 def validate_file_exists(file: Path, msg: str):
     if not file.exists():
         raise SystemExit(msg)
+
+
+def read_csv_by_line(file: Path) -> list:
+    lines = []
+    with open(file, 'r') as data:
+        for line in csv.DictReader(data):
+            lines.append(line)
+    return lines
+
+
+def get_app_specific_actions(file: Path) -> list:
+    app_specific_list = []
+    actions = read_csv_by_line(file)
+    for action in actions:
+        if bool(distutils.util.strtobool(action['App-specific'])):
```

Comment:
How action['App-specific'] looks in this context? `bool(distutils.util.strtobool(` looks a bit difficult.

Reply:
it is string `True`. We can do another function to parse str boolean, but with distutils this is build-in functionality, so from my point of view in this case better to use already created built-in functionality to parse string booleans.